CONTRIBUTING.md

```diff
@@ -1,29 +1,21 @@
 
-- Create a new Android library module inside the `plugins` project with Android Studio. Each plugin is a separate library that depends on the Mapbox Android SDK, and potentially on other third-party libraries. Besides the code for the plugin, make sure you include:
+- Create a new Android library module inside the `plugins` project with Android Studio. Each plugin is a separate library that depends on the MapLibre Android SDK, and potentially on other third-party libraries. Besides the code for the plugin, make sure you include:
 
   - Tests.
   - Javadoc that documents the plugin usage and purpose in detail.
 
-- Create a new activity inside the demo app that showcases how to use your plugin. As important as having a working plugin is to show how to use it so that anyone can include it in their projects easily.
-
-- Finally, once you're ready to share your code, list your plugin in this file and then open a PR for the `@mapbox/android` team to review.
+- Finally, once you're ready to share your code, list your plugin in this file and then open a PR for the `@maplibre/android` team to review.
 
 ## Adding or updating a localization
 
-The Mapbox Plugins SDK for Android features several translations contributed through [Transifex](https://www.transifex.com/mapbox/mapbox-plugins-android/). If your language already has a translation, feel free to complete or proofread it. Otherwise, please [request your language](https://www.transifex.com/mapbox/mapbox-plugins-android/) so you can start translating. Note that we’re primarily interested in languages that Android supports as system languages.
-
-While you’re there, please consider also translating the following related projects:
-
-* [OSRM Text Instructions](https://www.transifex.com/project-osrm/osrm-text-instructions/), which the Mapbox Directions API uses to generate textual and verbal turn instructions ([instructions](https://github.com/Project-OSRM/osrm-text-instructions/blob/master/CONTRIBUTING.md#adding-or-updating-a-localization)).
-* [Mapbox Navigation SDK for iOS](https://www.transifex.com/mapbox/mapbox-navigation-ios/), the analogous library for iOS applications ([instructions](https://github.com/mapbox/mapbox-navigation-ios/blob/main/CONTRIBUTING.md#adding-or-updating-a-localization)).
-* [Mapbox Maps SDK for Android](https://www.transifex.com/mapbox/mapbox-gl-native/), which is responsible for the map view and minor UI elements such as the Mapbox Telemetry permissions dialog
+The MapLibre Plugins SDK for Android features several translations contributed through [Transifex](https://www.transifex.com/maplibre/maplibre-plugins-android/). If your language already has a translation, feel free to complete or proofread it. Otherwise, please [request your language](https://www.transifex.com/maplibre/maplibre-plugins-android/) so you can start translating. Note that we’re primarily interested in languages that Android supports as system languages.
```

Comment:
We are not using Transifex at this point.

Reply:
Shall I delete the entire Transifex section ("Adding or updating a localization")?

Reply:
@tempranova Sounds good to me.